src/components/Contacts/ContactDetails/ContactDetailsTab/Other/ContactDetailsOther.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsTabQuery } from '../ContactDetailsTab.generated';
+
+const ContactOtherContainer = styled(Box)(({ theme }) => ({
+  margin: theme.spacing(1, 1, 1, 5),
+}));
+
+const ContactOtherTextContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  marginLeft: theme.spacing(2),
+}));
+
+const ContactOtherTextLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.hint,
+  marginRight: '5px',
+}));
+
+interface ContactDetailsOtherProp {
+  contact: ContactDetailsTabQuery;
+}
+
+export const ContactDetailsOther: React.FC<ContactDetailsOtherProp> = ({
+  contact,
+}) => {
+  const { t } = useTranslation();
+  const {
+    name,
+    preferredContactMethod,
+    locale,
+    timezone,
+    churchName,
+    website,
+  } = contact.contact;
+  return (
+    <Box>
+      <Typography variant="h6">{t('Other')}</Typography>
+      <ContactOtherContainer>
+        <ContactOtherTextContainer>
+          <ContactOtherTextLabel variant="subtitle1">
+            {t('Assignee')}
+          </ContactOtherTextLabel>
+          <Typography variant="subtitle1">{name}</Typography>
+        </ContactOtherTextContainer>
+        <ContactOtherTextContainer>
+          <ContactOtherTextLabel variant="subtitle1">
+            {t('Preferred Contact Method')}
+          </ContactOtherTextLabel>
+          <Typography variant="subtitle1">{preferredContactMethod}</Typography>
+        </ContactOtherTextContainer>
+        <ContactOtherTextContainer>
+          <ContactOtherTextLabel variant="subtitle1">
+            {t('Language')}
+          </ContactOtherTextLabel>
+          <Typography variant="subtitle1">{locale}</Typography>
+        </ContactOtherTextContainer>
+        <ContactOtherTextContainer>
+          <Typography variant="subtitle1">{timezone}</Typography>
+        </ContactOtherTextContainer>
+        <ContactOtherTextContainer>
+          <ContactOtherTextLabel variant="subtitle1">
+            {t('Church')}
+          </ContactOtherTextLabel>
+          <Typography variant="subtitle1">{churchName}</Typography>
+        </ContactOtherTextContainer>
+        <ContactOtherTextContainer>
+          <Link
+            href={website.startsWith('http') ? website : `http://${website}`}
```

Comment:
This seems a little odd that we just redirect out to random websites. Guess they should be entered but the current user though. Also a https redirect would be better but idk if that's practical.

I almost feel better about dumping the text on the page and letting the user copy/paste it into their url bar if they really want to visit the site. I feel (with no evidence) that this is a rarely used feature.

Reply:
@OzzieOrca I can go either way on this one.   This maybe something that Scott can answer later.  I will go ahead and remove it for now and we can add it back later.

Reply:
Sounds good. I didn't go and look at what the current behavior is.